### DISCUSSION TITLE
What is the correct API to use for a new integration?

### DISCUSSION THREAD

**Discussion by Melonking906** [FreshRSS/FreshRSS#6397] [Q&A] [answered]
Hello! I have been reading the documentation but I'm a little unclear about how to proceed.

I want to create a small NodeJS app that can:
- Add feeds to a category
- Remove feeds from a category
- Retrieve a list of feeds subscribed in a category
- Retrieve all articles from a feed
- Retrieve all articles from a category

What API is the correct one to work with when developing a new integration with FreshRSS, "Fever API" or "Google Reader API"? 

I have not been able to find any external documentation about either of these APIs outside of the FreshRSS documentation. Is the Greader API a legacy API? The documentation also repeatedly mentions that the API is for mobile use, but I'm not interested in mobiles, and I don't understand why it would be exclusive to mobiles. So a little guidance would be welcome!

**Comment by math-GH** [comment]
Lets dive a bit into the past to understand the Google Reader API:
In the early days of internet (in 2005) there was the Google Reader. It was very popular. But it was shut down about 10 years ago (in 2013). The API of the Google Reader became a de facto standard that is still used today. A lot of feed aggregators tried to have a similar API. So FreshRSS has it too.
Check the history: https://en.wikipedia.org/wiki/Google_Reader

Fever was also a feed aggregator and was stopped to develop.

I am sure you are able to find the documentation ;)

Of course you can use the API also for your NodeJS app.

**Comment by Alkarex** [comment, marked as answer]
I would not recommend using the Fever API for anything new. Our Google Reader API is likely the best option for what you are asking. See https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html
It is also used by several desktop clients, not only mobile apps: https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps
